junos/resource_security_nat_source_pool.go

```diff
@@ -65,6 +67,16 @@ func resourceSecurityNatSourcePool() *schema.Resource {
 				Optional:         true,
 				ValidateDiagFunc: validateNameObjectJunos([]string{}, 64, FormatDefault),
 			},
+			"pool_utilization_alarm_raise_threshold": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validation.IntBetween(50, 100),
+			},
+			"pool_utilization_alarm_clear_threshold": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validation.IntBetween(40, 100),
+			},
```

Comment:
Can you insert the new options to respect optional then alphabetical order ?
The `raise-threshold` is required with `clear-threshold` :
```suggestion
			"pool_utilization_alarm_raise_threshold": {
				Type:         schema.TypeInt,
				Optional:     true,
				ValidateFunc: validation.IntBetween(50, 100),
			},
			"pool_utilization_alarm_clear_threshold": {
				Type:         schema.TypeInt,
				Optional:     true,
				RequiredWith: []string{"pool_utilization_alarm_raise_threshold"},
				ValidateFunc: validation.IntBetween(40, 100),
			},
```